CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/components/TabContextMenuNGTest.java

```diff
@@ -60,19 +60,23 @@ public class TabContextMenuNGTest {
     private TabContextMenu tabContextMenu;
 
     @BeforeClass
+    //Causing issues with headless tests   
     public void setUpClass() throws Exception {
-        if (!FxToolkit.isFXApplicationThreadRunning()) {
-            FxToolkit.registerPrimaryStage();
-        }
+     
+//        if (!FxToolkit.isFXApplicationThreadRunning()) {
```

Comment:
I think this commented out code is needed as the toolkit needs to be enabled in order to work (it currently fails locally)